hack/arktos-cni.rc

```diff
@@ -134,32 +149,50 @@ ensure_devstack() {
 
 
 install_arktos_containerd() {
-    local CONTAINERD_DIR=$(dirname "$BASH_SOURCE[0]}")/../../arktos-containerd
-    [ -d ${CONTAINERD_DIR} ] || die "arktos-containerd repo not found. please clone futurewei-cloud/arktos-containerd branch wip-pod-annotation-to-cni-args side by side with arktos"
-    if [ ! -x ${CONTAINERD_DIR}/bin/containerd ]; then
-        echo "building arktos containerd..."
-        pushd ${CONTAINERD_DIR} > /dev/null
-        docker run --rm -e GOPATH=/go -v ${PWD}:/go/src/github.com/containerd/containerd -w /go/src/github.com/containerd/containerd golang:1.12 sh -c "apt update; apt install -y libbtrfs-dev libseccomp-dev; make bin/containerd"
-        popd > /dev/null
-        echo "...built"
-    fi
-
-    echo "make sure arktos containerd in power"
-    if [ ! "$(${CONTAINERD_DIR}/bin/containerd --version)" == "$(containerd --version)" ]; then
-        echo "stopping containerd shortly to replace the binary..."
-        local containerd_path=$(which containerd)
-        sudo systemctl stop containerd
-        local containerd_back_path=$(dirname ${containerd_path})/containerd_arktos_backup
-        sudo mv ${containerd_path} ${containerd_back_path}
-        sudo cp ${CONTAINERD_DIR}/bin/containerd ${containerd_path}
-        sudo systemctl start containerd
-        echo "warning: original containerd binary is saved to ${containerd_path})/containerd_arktos_backup"
-        echo "...done"
+#    local CONTAINERD_DIR=$(dirname "$BASH_SOURCE[0]}")/../../arktos-containerd
```

Comment:
Commented  code should not be there.

Reply:
Okay, we will revert the changes for this function.